packages/amplify-cli/src/lib/init-steps/s0-analyzeProject.js

```diff
@@ -123,7 +123,7 @@ async function getEnvName(context) {
   const isEnvNameValid = (inputEnvName) => {
     let valid = true;
 
-    if (inputEnvName.length > 10 || inputEnvName.length < 2 || /[^a-zA-Z]/g.test(inputEnvName)) {
+    if (inputEnvName.length > 10 || inputEnvName.length < 2 || /[^a-z]/g.test(inputEnvName)) {
```

Comment:
why only lower case support?

Reply:
because resource names like S3 don't support upper case. This is an artifact of this bug/issue -> https://github.com/aws-amplify/amplify-cli/issues/373#issuecomment-458689994

Reply:
OK, Can we have a separate commit for this. So its easier to track

Reply:
Yes, there is a separate commit for it -> https://github.com/aws-amplify/amplify-cli/pull/803/commits/6c581481c083c3af6f4c7a2eab489755b92ec4b4